### ISSUE TITLE
SecretChat Failed to Fetch update

### ISSUE THREAD

**Issue by ionwerz** [tdlib/td#341]
When I'm trying to crate new secret chat:
`"@type":"createNewSecretChat","user_id":4872874195`
I get this error:
```
[ 2][t 0][1534434758.576591492][SecretChatActor.cpp:870][!SecretChat 1947073279]	Failed to Fetch update: [Error : 0 : Unknown constructor [ID:0xaa48327d]]
[ 1][t 0][1534434758.576824903][SecretChatActor.cpp:1300][!SecretChat 1947073279]	INGORE MESSAGE: decryptedMessageLayer {
  random_bytes = bytes { }
  layer = 8
  in_seq_no = -1
  out_seq_no = -1
  message = decryptedMessageService8 {
    random_id = -2724851091046818866
    random_bytes = bytes { DF B1 73 A2 96 81 3A FC 77 48 D8 16 EF F4 1D }
    action = decryptedMessageActionNotifyLayer {
      layer = 73
    }
  }
}
```
This issue is only when I try to create secret chat to popular third party client for android "Plus Messenger"
When I create secret chat for official android Telegram client everything is good.
Few days ago there was no problem to create new secret chats for Plus messenger client. 
BTW, when I create new secret chat from Plus Messenger to TDlib, everything is ok.

**Comment by levlam** [comment]
Layer 8 support in secret chat, which is used in that message, was dropped very long ago. The other client should use decryptedMessageService from layer 17 instead. Nevertheless, support for this particular use case was recently added to TDLib, so it will possibly work after we will be able to update TDLib source code to the recent version.

**Comment by ionwerz** [comment]
But few days ago everything was OK, I was able to create chats for that third party client. Something was changed in TL scheme on telegram servers?

**Comment by levlam** [comment]
No. Secret chats are end-to-end and their schema fully depends on the client implementation. Possibly, it is somehow connected to updating of Plus Messenger source code.

**Comment by levlam** [comment]
The issue is supposed to be resolved in TDLib 1.3.0. Please reopen the issue providing additional details if it still happens to you.
